### PR TITLE
assignment 1 PR

### PR DESCRIPTION
**Pull request by Khedr05**

### CODE REVIEW

File: 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+TaskHandle_t LedTask_Handler = NULL;
+/*-----------------------------------------------------------*/
+
+void ledTask(void * pvParameters)
+{
+	for(;;)
+	{
+		GPIO_write(PORT_0 , PIN1 , PIN_IS_HIGH);
+
```
Comment (ahmedatef23) [inline 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c:94]: don't use magic number

File: 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/00_Task_One/main.c [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+{
+	for(;;)
+	{
+			if( btnStateSemaphore != NULL )
+      {
+       
+        if( xSemaphoreTake( btnStateSemaphore, ( TickType_t ) 100 ) == pdTRUE )
+        {
+            if(btnIsPressed == 1)
+						{
```
Comment (ahmedatef23) [inline 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/00_Task_One/main.c:125]: don't use magic number

File: 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/01_Task_Two/main.c [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+
+void sendString100msTask(void * pvParameters)
+{
+	for(;;)
+	{
+		 if( uartSemaphore != NULL )
+      {
+       
+        if( xSemaphoreTake( uartSemaphore, ( TickType_t ) 100 ) == pdTRUE )
```
Comment (ahmedatef23) [inline 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/01_Task_Two/main.c]: don't use magic number

File: 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/00_Task_One/main.c (hunk 2 above)
Comment (Khedr05) [reply 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/00_Task_One/main.c:125]: DONE

File: 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/01_Task_Two/main.c (hunk 3 above)
Comment (Khedr05) [reply 08_Sprint_thirteen/S_RTOS_02_Interprocess_Communication/01_Task_Two/main.c]: DONE

File: 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c (hunk 1 above)
Comment (Khedr05) [reply 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c:94]: Please mention line number i cant see magic numbers in this lines PORT_0 , PIN1 and PIN_IS_HIGH all of them is macro in gpio config file thanks

Comment (ahmedatef23) [reply 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c:94]: in line number 94 and 98

Comment (Khedr05) [reply 08_Sprint_thirteen/S_RTOS_01_Introduction_To_Freertos/00_Task_One/main.c:94]: okay i change it and change it in all other files now all tasks not contain magic numbers not this only thanks

**Pull request merged**